server/models.py

```diff
@@ -0,0 +1,65 @@
+from app import db
+from sqlalchemy.sql import func
+from sqlalchemy import ForeignKey
+
+class User(db.Model):
+    __tablename__ = 'user'
+
+    id = db.Column(db.Integer, primary_key=True)
+    username = db.Column(db.String())
+    email = db.Column(db.String())
+    password = db.Column(db.String())
+    created_time = db.Column(db.DateTime, default=func.utcnow())
+
+    def __init__(self, id, email, password, created_time):
+        self.id = id
+        self.email = email
+        self.password = password
+        self.created_time = created_time
+
+class Contest(db.Model):
+    __tablename__ = 'contest'
+
+    id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String())
+    description = db.Column(db.String())
+    prize_contest = db.Column(db.Integer())
+    deadline_date = db.Column(db.DateTime)
+    created_time = db.Column(db.DateTime, default=func.utcnow())
+    update_time = db.Column(db.DateTime, onupdate=func.utcnow()) # store updated time of row
+    contest_creater = db.Column(db.Integer, ForeignKey('contest_user.id'))
+
+    def __init__(self, id, title, description, prize_contest, deadline_date, created_time, update_time, contest_creater):
```

Comment:
created_time should also get autogenerated on init and won't be required to pass in